opennebula/resource_opennebula_image.go

```diff
@@ -635,6 +699,11 @@ func generateImageTemplate(d *schema.ResourceData) (string, error) {
 		tpl.Add(imk.Target, val.(string))
 	}
 
+	tagsInterface := d.Get("tags").(map[string]interface{})
+	for k, v := range tagsInterface {
+		tpl.AddPair(strings.ToUpper(k), v)
```

Comment:
AddPair already use ToUpper function internally.
But, Del, and other GetPair etc... functions does not.

We may need to fix this in Goca later, as the behaviour is not consistent for all methods...

Reply:
yes. But it not  linked directly to this patch ;)